ROADMAP.md

```diff
@@ -1,30 +1,32 @@
 
-### How to help?
+## How to help?
 Discussion on the roadmap can take place during [community meetings](https://carvel.dev/community/). If you want to provide suggestions, use cases, and feedback to an item in the roadmap, please add them to the [meeting notes](https://hackmd.io/F7g3RT2hR3OcIh-Iznk2hw) and we will discuss them during community meetings. Please review the roadmap to avoid potential duplicated effort.
 
-### How to add an item to the roadmap?
+## How to add an item to the roadmap?
 One of the most important aspects in any open source community is the concept of proposals. Large changes to the codebase and / or new features should be preceded by a [proposal](https://github.com/vmware-tanzu/carvel-community/tree/develop/proposals) in our repo.
 For smaller enhancements, you can open an issue to track that initiative or feature request.
 We work with and rely on community feedback to focus our efforts to improve Carvel and maintain a healthy roadmap.
 
-### Current Roadmap
+## Current Roadmap
 The following table includes the current roadmap for Carvel. If you have any questions or would like to contribute to Carvel, please attend a [community meeting](https://carvel.dev/community/) to discuss with our team. If you don't know where to start, we are always looking for contributors that will help us reduce technical, automation, and documentation debt.
 Please take the timelines & dates as proposals and goals. Priorities and requirements change based on community feedback, roadblocks encountered, community contributions, etc. If you depend on a specific item, we encourage you to attend community meetings to get updated status information, or help us deliver that feature by contributing to Carvel.
 
-|Theme|Description|Timeline|
-|---|---|---|
-| Package Author Improvements | **[ytt]** [Schema Validations](https://hackmd.io/pODV3wzbT56MbQTxbQOOKQ#Part-7-Validating-Documents) Experimental: Configuration authors can specify the valid range or format of the data values. | Late July |
-| Provide Stability | **[kapp-controller]** [Surface namespaces for deployed resources](https://github.com/vmware-tanzu/carvel-kapp-controller/issues/535) | Early August |
-| Provide Stability | **[kapp-controller]** Version safeguards: To provide safeguards and visibility for kapp-controller and kubernetes versions. | Early August |
-| Package Author Improvements | **[kctrl]** kctrl commands for package authors. | Aug 2022 |
-| Package Author Improvements | **[ytt]** [Schema Validations](https://hackmd.io/pODV3wzbT56MbQTxbQOOKQ#Part-7-Validating-Documents) GA: Configuration authors can specify the valid range or format of the data values. | Late August |
-| Provide Stability | **[kapp-controller]** Minimize registry communications: To provide a more performant experience for users leveraging large PackageRepos. | Late August |
-
+`Last Updated: Aug 2022`
+|Theme|Feature|Stage|Timeline|
```

Comment:
Is there a definition of the Stages somewhere? I'm curious what all of the options could be.